intake/catalog/entry.py

```diff
@@ -74,7 +74,7 @@ def __call__(self, persist=None, **kwargs):
                 'always', 'never', 'default']:
             raise ValueError('Persist value (%s) not understood' % persist)
         persist = persist or self._pmode
-        s = self.get(**kwargs)
+        s = self.get(**kwargs)  # Should this live in an `else` below the `if`?
```

Comment:
We need an instance of `s`, with all its parameters, to know if a persisted version exists.